Common/src/DefaultRawMemoryAllocator.cpp

```diff
@@ -39,6 +39,37 @@
 namespace Diligent
 {
 
+namespace
+{
+
+#if PLATFORM_ANDROID && __ANDROID_API__ < 28
+void* AllocateAlignedFallback(size_t Size, size_t Alignment)
```

Comment:
This looks good to me.
I made some refactoring to have only one `#if` condition (right now `#if PLATFORM_ANDROID && __ANDROID_API__ < 28` is repeated three times)
Can you rebase on the latest and update the branch?